test/runtest.py

```diff
@@ -1511,6 +1511,26 @@ def wait_for_object_table():
 
     ray.worker.cleanup()
 
+  def testTaskProfileAPI(self):
+    ray.init(redirect_output=True)
+
+    @ray.remote
+    def say_hi():
+      print("hi")
```

Comment:
Let's call this `f` and not have it print anything since we aren't using the output of the print.